UnrealPlugin/README.md

```diff
@@ -268,6 +272,16 @@ EXPOSE 7777/udp
 WORKDIR /server
 
 COPY --chown=ue:ue . /server
-CMD ./<projectname>Server.sh
+USER root
+CMD ./startup.sh
 ```
-Thank you to [narthur157](https://github.com/narthur157) for this amazing [Dockerfile](https://github.com/narthur157/playfab-gsdk-ue4).
\ No newline at end of file
+
+### startup&#46;sh bash script:
+```bash
+logFolderName=$(cat $GSDK_CONFIG_FOLDER | jq .logFolder)
```

Comment:
instead of installing `jq` and doing this to get the log folder name, any chance we can grab the `PF_SERVER_LOG_DIRECTORY` ([source](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/basics-of-a-playfab-game-server#server-scripts-and-playfab-configured-environment-variables))?
I assume that this won't work since we need to chance the owner of the directory to the Unreal user, but would like to get it validated.

Reply:
I completely forgot about that environment variable, i have tested it, and it works with it directly, so i have removed the json parsing library from teh docker file again.